### DISCUSSION TITLE
Release #5360 (invoking code-server in integrated terminal) asap

### DISCUSSION THREAD

**Discussion by jeengbe** [coder/code-server#5421] [General] [answered]
9087e0c0915096e67d6d3221825874f2a562be0f fixes a bug with invoking code-server in integrated terminal, which I personally would like to see fixed as soon as possible.

Simply react with 👍🏻👎🏻, depending on whether you'd prioritize this issue as well, and release it asap :)

**Comment by jsjoeio** [comment, marked as answer]
Thanks @jeengbe!

**Reply by jeengbe** [reply]
I have to thank 🙏🏻
